psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -65,15 +66,31 @@ public String getComment() {
         return "PSS/E Format to IIDM converter";
     }
 
+    private String findExtension(ReadOnlyDataSource dataSource, boolean throwException) throws IOException {
+        for (String ext : EXTS) {
+            if (dataSource.exists(null, ext)) {
+                return ext;
+            }
+        }
+        if (throwException) {
+            throw new PsseException("File " + dataSource.getBaseName()
+                    + "." + Joiner.on("|").join(EXTS) + " not found");
```

Comment:
Use `String.join("|", EXTS)` instead of `Joiner.on`?

Reply:
I modified PSSE and UCT import since they werer using the same approach